base/threadcall.jl

```diff
@@ -44,7 +44,7 @@ macro threadcall(f, rettype, argtypes, argvals...)
     body = wrapper.args[2].args
     args = Symbol[]
     for (i,T) in enumerate(argtypes)
-        arg = symbol("arg$i")
+        arg = Symbol("arg$i")
```

Comment:
Makes sense, but I would have thought the constructors created when renaming `symbol()` to `Symbol()` would have been enough.